version.json

```diff
@@ -1 +1 @@
-{"version":"4.3.1","channel":"stable","repositories":[{"url":"http:\/\/api.blocklandglass.com\/api\/2\/repository.php","format":"JSON","id":"11"},{"url":"http:\/\/cdn.blocklandglass.com\/repository.txt","format":"JSON","id":"11"}]}
\ No newline at end of file
+{"version":"4.3.2","channel":"stable","repositories":[{"url":"http:\/\/api.blocklandglass.com\/api\/2\/repository.php","format":"JSON","id":"11"},{"url":"http:\/\/cdn.blocklandglass.com\/repository.txt","format":"JSON","id":"11"}]}
```

Comment:
I'm generally pretty lazy with this because it's put in when we upload to the mod manager anyway. Honestly might as well add it to .gitignore